app/views/layouts/application.html.erb

```diff
@@ -1,45 +1,52 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Podcastr</title>
+  <title>ρodcastr</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <%= stylesheet_link_tag 'application', media: 'all' %>
   <%= csrf_meta_tags %>
+  <link href="https://fonts.googleapis.com/css?family=Play&display=swap" rel="stylesheet">
```

Comment:
this should be imported from the stylesheet

Reply:
Moved to importing font into `application.css`